lite/examples/posenet/android/app/src/main/java/org/tensorflow/lite/examples/posenet/PosenetActivity.kt

```diff
@@ -570,6 +549,17 @@ class PosenetActivity :
       paint
     )
 
+//    val rt = Runtime.getRuntime()
+//    canvas.drawText(
+//      "Native/JVM: %.2f/%.2f GB".format(
+//        Debug.getNativeHeapSize() / (1024 * 1024 * 1024.0),
+//        (rt.totalMemory() - rt.freeMemory()) / (1024 * 1024 * 1024.0)
+//      ),
+//      (15.0f * widthRatio),
+//      (90.0f * heightRatio + bottom),
+//      paint
+//    )
```

Comment:
Can you delete these lines?

Reply:
I removed the commented code. I couldn't find any more unused imports.